### PR TITLE
[RELEASE] learnosity-sdk-java v0.16.0

### PR DESCRIPTION
**Pull request by flakeparadigm**
- Changelog updated to match PHP SDK and https://keepachangelog.com/en/1.0.0/
- Readme updated to bump references of v0.15 to v0.16.0
- Update JARs in the repo with new code (These were created running `vgutil run 'mvn package'`)

### CODE REVIEW

**Comment by shtrom** [comment]
Looks good, but can we move all the built Jar files from the repo, and attach them to the release, after creation (i.e., tagging), in https://github.com/Learnosity/learnosity-sdk-java/releases.

**Comment by flakeparadigm** [comment]
@shtrom I'd rather not make that kind of change right now (like updating info in the readme, adding Dist to gitignore, etc) because we'd like to get this out and done without working out our future build/release process. However I'm happy to add them to the release when I tag after this is merged.

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Versioning](http://semver.org/spec/v2.0.0.html).
 * Upgrade dependency `org.apache.httpcomponents:httpclient` to v4.5.8 to patch vulnerabilities.
 
 ### Fixed
-* Fixed an issue where DataAPI's `requestRecursive` method would throw an exception upon receiving a response from Data API endpoints that set the "data" field of the response to an object (like the `itembank/questions` endpoint when `item_references` is included in the request).
+* Fixed an issue where Data API's `requestRecursive` method would throw an exception upon receiving a response from Data API endpoints that set the "data" field of the response to an object (like the `itembank/questions` endpoint when `item_references` is included in the request).
```
Comment (shtrom) [inline CHANGELOG.md]: I wonder if we need `DataAPI` here, as I thing @flakeparadigm was refering to the actual Java class.

Comment (flakeparadigm) [reply CHANGELOG.md]: Yeah, Edwin and I are already discussing that.

Comment (flakeparadigm) [reply CHANGELOG.md]: (Though my original capitalization was also incorrect 😅 haha)

**Comment by flakeparadigm** [comment]
Squashed and rebased for merge.

**Pull request merged**